projects/dotcms/src/lib/api/DotApiPage.ts

```diff
@@ -32,7 +35,7 @@ export class DotApiPage {
 
         params = {
             ...params,
-            url: `/api/v1/page/json${params.url}`
+            url: `/api/v1/page/${format}${params.url}`
```

Comment:
test?

Reply:
Added in 458afa2. However, I wasn't able to actually run the tests. Is it possible to only run the tests for the `dotcms` project?